### PR TITLE
[tw] Add content/zh-tw/pod.md

### LINKED ISSUE #2277: [tw] Localize Pod for Traditional Chinese
1. Localize Pod for Traditional Chinese
2. https://glossary.cncf.io/pod/

### PR DESCRIPTION
**Pull request by hwchiu**
### Describe your changes
Localize the Pod to zh-tw language
Origin content from https://glossary.cncf.io/pod/

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2277 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/pod.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+如果將這些密切相關的容器單獨管理，則會導致冗余的管理任務出現。
+舉例來說，操作者必須反覆地確保這些容器彼此的部署狀態以確保他們保持在一起。
+此外這些密切相關的容器其生命週期必須一致同步的被處理，但是卻只能各別單獨管理。
+
+## 如何幫助 {#how-it-helps}
+
+Pods 將緊密關係的容器們打包成一個單一的單位，此舉大幅度的簡化了容器操作。
+譬如，輔助用容器通常都會伴隨主要容器一起使用已加入額外功能或是設定全域設定。
+範例包含將基本設定給注入或是套用到主要容器的容器，
+以並行模式來處理網路流量與路由相關的模式(參見 [服務網格](/zh-tw/service-mesh/))，
```
Comment (ydFu) [inline content/zh-tw/pod.md:29]: EN
```
_sidecar_ (containers) that handle network traffic routing for the main container (see [service mesh](/service-mesh/)),
```

I refer to the traditional Chinese of other cloud providers, and the `sidecar` does not need translation now

File: content/zh-tw/pod.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+雖然容器通常作為獨立的單位運行以及控制特定的工作負載，
+但是某些情況下，容器間需要以緊密耦合的方式進行互動與被控制。
+
+如果將這些密切相關的容器單獨管理，則會導致冗余的管理任務出現。
+舉例來說，操作者必須反覆地確保這些容器彼此的部署狀態以確保他們保持在一起。
+此外這些密切相關的容器其生命週期必須一致同步的被處理，但是卻只能各別單獨管理。
+
+## 如何幫助 {#how-it-helps}
+
+Pods 將緊密關係的容器們打包成一個單一的單位，此舉大幅度的簡化了容器操作。
```
Comment (ydFu) [inline content/zh-tw/pod.md:26]:
```suggestion
Pod 將緊密關係的容器們打包成一個單一的單位，此舉大幅度的簡化了容器操作。
```
Traditional Chinese does not have plural nouns, it is better to change them to singular nouns

Comment (hwchiu) [reply content/zh-tw/pod.md]: Done

File: content/zh-tw/pod.md (hunk 1 above)
Comment (hwchiu) [reply content/zh-tw/pod.md]: Done

File: content/zh-tw/pod.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: Pod
+status: Completed
+category: 概念
+tags: ["基礎設施", "基本原理", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+Pod 是 Kubernetes 環境中最基本的可部署單位。
```
[the remark this answers is not in the document]
Comment (hwchiu) [reply content/zh-tw/pod.md]: Done

File: content/zh-tw/pod.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+它代表部署和管理容器化應用程式的必要組件。
+每個 Pod 包含一個應用程式實例，並且可以由一個或是多個[容器](/zh-tw/container/) 所組成。
+Kubernetes 將 Pod 視為大規模部署的一部分並且可根據需求[垂直](/zh-tw/vertical-scaling/)或是[水平](/zh-tw/horizontal-scaling/)地擴展 Pod 的規模
```
[the remark this answers is not in the document]
Comment (hwchiu) [reply content/zh-tw/pod.md]: Done

**Review by ydFu — APPROVED** [review]
LGTM,thank @hwchiu

**Review by pichuang — APPROVED** [review]
LGTM, and please rebase the 2 commits into 1 commit before merge

**Pull request merged**